### ISSUE TITLE
SchemaError when converting Wyscout events to SPADL actions

### ISSUE THREAD

**Issue by BWyckaert** [ML-KULeuven/socceraction#157]
When using the ´convert_to_actions´ method 

https://github.com/ML-KULeuven/socceraction/blob/5f6dcb5d2ed0d12b599b9719c0e1fef21125212e/socceraction/spadl/wyscout.py#L25

to convert Wyscout events to SPADL actions, I get the following error:

![image](https://user-images.githubusercontent.com/58517640/148637533-8dd8e4e9-d1ee-4864-9ee8-99303dfbdfda.png)

I am using Python 3.8 and socceraction 1.1.3. The bug/error can be recreated as follows (set the download flag as needed):

```python
    pwl = socceraction.data.wyscout.PublicWyscoutLoader(download=True)
    competitions = pwl.competitions()
    world_cup = competitions.loc[6]
    wc_games = pwl.games(world_cup.competition_id, world_cup.season_id)
    game_id = wc_games.at[0, 'game_id']
    home_team_id = wc_games.at[0, 'home_team_id']
    events = pwl.events(game_id)
    actions = socceraction.spadl.wyscout.convert_to_actions(events, home_team_id)
```

Removing the `.astype(int)` on line 53 seems to fix this for me.

https://github.com/ML-KULeuven/socceraction/blob/5f6dcb5d2ed0d12b599b9719c0e1fef21125212e/socceraction/spadl/wyscout.py#L53

**Comment by probberechts** [comment]
Are you perhaps running Windows? I did not know this either, but apparently `int` is interpreted as `int64` on Linux and as `int32` on Windows. You can read more about it in these issues:
- https://github.com/pandas-dev/pandas/issues/44925
- https://github.com/numpy/numpy/issues/9464

Could you check if adding `coerce = True` to the [`SPADLSchema`](https://github.com/ML-KULeuven/socceraction/blob/5f6dcb5d2ed0d12b599b9719c0e1fef21125212e/socceraction/spadl/schema.py#L10) config fixes it? I do not have a Windows machine, so I cannot check it myself.

**Comment by BWyckaert** [comment]
I am running Windows, yes. Adding `coerce = True` does fix the issue, but as the line where I proposed the change is part of a for loop you would have to add `coerce = True` to some other columns as well, as they all suffer from the same problem, namely: `action_id`, `period_id`, `bodypart_id`, `type_id` and `result_id`. As I'm also experimenting with atomic SPADL and VAEP, I noticed that you would also have to add `coerce = True` to `period_id` in [`AtomicSPADLSchema`](https://github.com/ML-KULeuven/socceraction/blob/5f6dcb5d2ed0d12b599b9719c0e1fef21125212e/socceraction/atomic/spadl/schema.py#L10)

**Comment by probberechts** [comment]
Thanks for your help @BWyckaert! This should be fixed in socceraction v1.2.1, which will appear on PyPi in a couple of minutes.

I've also created an upstream issue in the Pandera repo: 
- https://github.com/pandera-dev/pandera/issues/726
